Ktisis/Overlay/Gizmo.cs

```diff
@@ -19,11 +22,14 @@ public class Gizmo {
 
 		public OPERATION? ForceOp = null;
 
-		// Compose & Decompose
-		// Compose updates the matrix using given values.
-		// Decompose retrieves values from the matrix.
+		public bool IsUsing = ImGuizmo.IsUsing();
```

Comment:
Should this boolean be live? Right now it only records the state of the gizmo at time of class construction.

Reply:
Leaving that here was a mistake. Most likely a leftover from a previous previous.